cli/args/mod.rs

```diff
@@ -467,8 +475,8 @@ impl CliOptions {
       .unwrap_or(false)
   }
 
-  pub fn location_flag(&self) -> Option<&Url> {
-    self.flags.location.as_ref()
+  pub fn location_flag(&self) -> &Option<Url> {
+    &self.flags.location
```

Comment:
I think for outputs `&Option<T>` makes sense if able and for inputs `Option<&T>` (I initially wrote this code here)